nav2-usability/scenario-code/nested-routing/nested_routing_vrouter.dart

```diff
@@ -0,0 +1,174 @@
+// Copyright 2021, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Nested example
+/// Done using VRouter
+
+import 'package:flutter/material.dart';
+import 'package:vrouter/vrouter.dart';
+
+void main() {
+  runApp(BooksApp());
+}
+
+class BooksApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return VRouter(
+      initialUrl: '/books/all',
+      routes: [
+        VNester(
+          path: null,
+          widgetBuilder: (child) => AppScreen(child: child),
+          nestedRoutes: [
+            VWidget(
+              path: '/books/all',
+              aliases: ['/books/new'],
+              // We don't want an animation between path and alias so we use a constant key
+              key: ValueKey('books'),
+              widget: Builder(
+                builder: (context) => BooksScreen(
+                  initialSelectedTab: context.vRouter.url!.contains('/new') ? 0 : 1,
+                ),
+              ),
+              buildTransition: (animation, _, child) =>
+                  FadeTransition(opacity: animation, child: child),
+            ),
+            VWidget(
+              path: '/settings',
+              widget: SettingsScreen(),
+              buildTransition: (animation, _, child) =>
+                  FadeTransition(opacity: animation, child: child),
+            ),
+          ],
+        ),
+      ],
+    );
+  }
+}
+
+class AppScreen extends StatelessWidget {
+  final Widget child;
+
+  const AppScreen({Key? key, required this.child}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: child,
+      bottomNavigationBar: BottomNavigationBar(
+        currentIndex: context.vRouter.url!.contains('/books') ? 0 : 1,
+        onTap: (idx) {
+          if (idx == 0) {
+            context.vRouter.push('/books/all');
```

Comment:
this should be /books/new

Reply:
Thanks! Done in faa5f92
I also changed the initial url to `/books/new`